sdk/test/localnet.test.ts

```diff
@@ -0,0 +1,677 @@
+// Copyright (c) Mysten Labs, Inc.
+// Modifications Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Transaction } from '@iota/iota-sdk/transactions';
+import {
+    NANOS_PER_IOTA,
+    // normalizeIotaAddress
+} from '@iota/iota-sdk/utils';
+import { beforeAll, describe, expect, it } from 'vitest';
+
+import { ALLOWED_METADATA, IotaNamesClient, IotaNamesTransaction } from '../src/index.js';
+import { getClient, setupIotaClient, TestToolbox } from './toolbox';
+
+const LOCALNET_PRICING = {
+    THREE_LETTER: { registration: 0.5, renewal: 0.15 },
+    FOUR_LETTER: { registration: 0.1, renewal: 0.05 },
+    FIVE_PLUS_LETTER: { registration: 0.01, renewal: 0.005 },
+} as const;
+
+export const e2eLocalnetDryRunFlow = async (toolbox: TestToolbox) => {
+    // For localnet, we use the getClient helper from toolbox
+    const client = getClient();
+
+    const sender = toolbox.address();
+    const iotaNamesClient = new IotaNamesClient({
+        client,
+        network: 'localnet',
```

Comment:
Here instead of passing localnet, just pass he package info directly
```suggestion
        packageInfo: localnetPackageInfo
```